albumentations/augmentations/transforms.py

```diff
@@ -794,6 +795,89 @@ def get_transform_init_args_names(self):
         return ("max_part_shift",)
 
 
+class RandomCropFromBorders(DualTransform):
+    """Crop bbox from image randomly cut parts from borders without resize at the end
+
+    Args:
+        crop_value (float): float value in (0.0, 0.5) range. Default 0.1
+        crop_left (float): float value in (0.0, 1.0) range. Default 0.1
+        crop_right (float): float value in (0.0, 1.0) range. Default 0.1
+        crop_top (float): float value in (0.0, 1.0) range. Default 0.1
+        crop_bottom (float): float value in (0.0, 1.0) range. Default 0.1
```

Comment:
Please, add description that values randomly chooses from ranges `[0, crop_left * width)`, `[width - crop_right * width, width)` and etc.